docs/pages/rigid-object-tutorial.rst

```diff
@@ -0,0 +1,154 @@
+:ref-prefix:
+    habitat_sim.simulator
+    habitat_sim.sim
+    habitat_sim.agent
+    habitat_sim.attributes
+
+Interactive Rigid Objects
+#########################
+
+:summary: This tutorial demonstrates rigid object interactions in Habitat-sim -- instancing, dynamic simulation, and kinematic manipulation.
+
+.. contents::
+    :class: m-block m-default
+
+First, download the `example objects`_ and extract them into data/objects/.
+
+The example code below is runnable via:
+
+.. code:: shell-session
+
+    $ python examples/tutorials/rigid_object_tutorial.py
+
+.. _example objects: http://dl.fbaipublicfiles.com/habitat/objects_v0.1.zip
+
+Import necessary modules, define some convenience functions, and initialize the :ref:`Simulator` and :ref:`Agent`.
+
+.. include:: ../../examples/tutorials/rigid_object_tutorial.py
+    :code: py
+    :start-after: # [setup]
+    :end-before: # [/setup]
+
+.. include:: ../../examples/tutorials/rigid_object_tutorial.py
+    :code: py
+    :start-after: # [initialize]
+    :end-before: # [/initialize]
+
+`Simulation Quickstart`_
+========================
+
+Basic rigid body simulation can be achieved by simply loading a template, instancing an object, and stepping the physical world.
+In this example, a sphere object template is loaded and the object is instanced in the scene above the table.
+When the simulation is stepped, it falls under the force of gravity and reacts to collisions with the scene.
+
+.. include:: ../../examples/tutorials/rigid_object_tutorial.py
+    :code: py
+    :start-after: # [basics]
+    :end-before: # [/basics]
+
+.. image:: images/rigid-object-tutorial-images/sim_basics.gif
```

Comment:
Gif looks very cool. But the video at my end doesn't have the sphere moving. Likely the same issue at #613. 

![sim_basics2](https://user-images.githubusercontent.com/2941091/80855969-bac85f00-8bfa-11ea-9616-21bfaff79dc2.gif)

Reply:
@dhruvbatra Did you install Bullet and compile with `--bullet`? If not there will not be any dynamics. You should see an indication in the console output that bullet was not available.

Reply:
Ah, that might be it. Let me try again.

Reply:
Following up. I am rebuilding with bullet but still seeing errors like these: 

```bash
(habitat)
dbatra at dbatra-mbp in ~/projects/habitat-sim on rigid-object-tutorial [?]
$ rm -rf build/
(habitat)
dbatra at dbatra-mbp in ~/projects/habitat-sim on rigid-object-tutorial [?]
$ python setup.py install --bullet
...
<successful>
...
dbatra at dbatra-mbp in ~/projects/habitat-sim on rigid-object-tutorial [?]
$ python examples/tutorials/rigid_object_tutorial.py
...
---
Physics was enabled and Bullet physics engine was specified, but the project is built without Bullet support. Objects added to the scene will be restricted to kinematic updates only. Reinstall with --bullet to enable Bullet dynamics.
---
```

I don't believe this error message because the viewer works: 
```bash
(habitat)
dbatra at dbatra-mbp in ~/projects/habitat-sim on rigid-object-tutorial [?]
$ ./build/viewer --enable-physics data/scene_datasets/habitat-test-scenes/van-gogh-room.glb
```

I am able to add object with "o". And I see them colliding.

Reply:
This sounds like it could be an issue with python module building. @erikwijmans does this seem possible?

Reply:
Just following up on this -- I'm not sure what changed, but after a clean clone and build, this example is working for me, so we're ready to go. 

One minor thing -- I'm seeing a slightly different collision in `dynamic_control.mp4` (attached) than what you uploaded. I suspect it's something to do with system-specific initial conditions? 

![dynamic_control](https://user-images.githubusercontent.com/2941091/81492435-749f7b00-924c-11ea-8a98-37b28c8f101d.gif)

Reply:
It looks like your box object's COM is off. I modified this somewhat recently. Be sure to download the updated objects and replace them in the data/objects folder. (same link).

The gifs were also out of date, I've updated them for the new version. Try it now 😄

Reply:
Yes, can confirm, it was the assets. After pulling to latest on this branch:

with my old objects: 
![dynamic_control](https://user-images.githubusercontent.com/2941091/81630322-21e3d180-93ba-11ea-8139-faf424ccc7f3.gif)

with new objects:
![dynamic_control2](https://user-images.githubusercontent.com/2941091/81630362-3fb13680-93ba-11ea-9d86-0cbf4b8554f2.gif)

Resolved.